pony-sync.md

```diff
@@ -0,0 +1,35 @@
+# Pony Sync
+
+The Pony Sync is a weekly meeting where Pony contributors discuss topics related to Pony, including recently added or modified issues.
+
+## Running the Pony Sync
+
+The Pony Sync takes place as a [Zoom](https://zoom.us) call. In order to run the Pony Sync you must have Zoom installed, and be logged in as `ponylang.main@gmail.com`. A list of recently modified issues can be generated using the [pony-sync-helper](https://github.com/ponylang/pony-sync-helper) application; this list should be pasted into the `sync` stream in Zulip, in a topic whose name is the date of the meeting in US date format (for example May 12, 2020 would be "5/12/2020").
+
+### Prerequisites
+
+In order to run the Pony Sync meeting you will need:
+
+1. A GitHub account
+2. A GitHub personal access token for running the `pony-sync-helper` program
+3. The [`pony-sync-helper`](https://github.com/ponylang/pony-sync-helper) program
+4. [Zulip](https://zulipchat.com/) and a Zulip account
+5. A 1Password account that is connected to the Pony 1Password account
+6. [Zoom](https://zoom.us)
+7. The Zoom password for the `ponylang.main@gmail.com` account, which is available in the Pony 1Password vault
```

Comment:
Should we include a link to the 1Password vault here? I have sometimes forgotten the correct URL when working from a different computer.

Reply:
Added a link above to the item that mentions the 1password account.